core/whistle_media-1.0.0/src/wh_media_map.erl

```diff
@@ -362,7 +362,10 @@ get_map(?WH_MEDIA_DB = Db, PromptId) ->
             #media_map{id=MapId
                        ,account_id=Db
                        ,prompt_id=PromptId
-                       ,languages=wh_json:new()
+                       ,languages=wh_json:set_value(wh_media_util:default_prompt_language()
+                                                    ,<<"/", ?WH_MEDIA_DB/binary, "/", (wh_media_util:default_prompt_language())/binary, "%2F", PromptId/binary>>
```

Comment:
This should probably be `wh_media_util:prompt_path(?WH_MEDIA_DB, PromptId)` instead of building the binary. See if that works and update the pull request if so. Otherwise, this looks good!